.travis.yml

```diff
@@ -1,8 +1,28 @@
-language: java
-
 matrix:
   include:
+    - name: deploy markbind
+      language: node
+      node_js:
+        - '8'
+      install:
+        - npm i -g markbind-cli
+      before_script: cd docs
+      script: markbind build && markbind deploy --travis
+      branches:
+        only:
+        - master
+      deploy:
+        provider: pages
+        skip_cleanup: true
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        keep_history: true
+        local_dir: _site
+        on:
+          branch: master
+          repo: reposense/RepoSense
```

Comment:
This line means the deployment will run only on reposense/RepoSense, and only on master branch. it will not run on your fork. 
If you want to test this on your fork, 
* remove this line. Also,
* set up travis for your fork
* set up the GITHUB_TOKEN variable for your fork

Reply:
Thanks for guiding Prof! All seems okay.

Reply:
Can try to test this on your fork to see if it works?

Reply:
Tried on fork. Pages get deployed.

Reply:
Can provide the travis build link and the deployed gh pages link? Wanted to see the behaviour of the build.

Reply:
The deployed site is in this [link](https://tejas2805.github.io/RepoSense/)

Reply:
Didn't realise, it doesn't show the MarkBind features. Trying to figure out why.

Reply:
The travis build is at this [link](https://travis-ci.com/github/Tejas2805/RepoSense/jobs/318569240)

Reply:
The site has been successfully deployed using Netlify. Can try this [link](https://reverent-einstein-88045a.netlify.com/userguide)

Reply:
> The site has been successfully deployed using Netlify. Can try this link

Hmm I thought we are deploying to gh pages instead?

Reply:
Yup. Saying it to mention, that site is alright. Need to figure out the travis code.

Reply:
Perhaps u can refer [here](https://markbind-master.netlify.com/userguide/deployingthesite#configuring-your-repository-in-travis-ci)

Reply:
Done. @jamessspanggg Check this [link](https://tejas2805.github.io/RepoSense/)

Reply:
Looks good. Can update the code in this PR to reflect the changes in ur forked?

Reply:
> Done. @jamessspanggg Check this link

@Tejas2805 it seems like the link doesn't work, when you click on the any of the links on the nav bar, it shows a 404 not found error.